CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ and apply all learnings to your own projects.
 
 When reporting issues please include as much detail as possible about your operating system, Nebari version, and dependencies version.
 Whenever possible, also include a brief, self-contained code example that demonstrates the problem.
-This [blog post by Matthew Rockling](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) is a good primer on how to craft minimal bug reports.
+This [blog post by Matthew Rocklin](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) is a good primer on how to craft minimal bug reports.
```

Comment:
I don't think we need to manually make updates to this. It syncs directly from GitHub: https://github.com/nebari-dev/.github/blob/main/CONTRIBUTING.md

Reply:
Ok, I'll suggest the correction over there.

Reply:
[#27](https://github.com/nebari-dev/.github/pull/27)